src/work-allocation-2/components/task-manager-filter/task-manager-filter.component.ts

```diff
@@ -127,16 +137,16 @@ export class TaskManagerFilterComponent implements OnInit, OnDestroy {
       name: 'role',
       options: [
         {
-          key: 'Judicial',
-          label: 'Judicial'
+          key: PersonRole.JUDICIAL,
```

Comment:
I would rather the key was not defined as such because it is actually more similar to RoleCategory than PersonRole - e.g. data comes back as 'LEGAL_OPERATIONS'